### PR TITLE
Draggable

### PR DESCRIPTION
**Pull request by SlothSan**

### CODE REVIEW

File: src/Component/Applications/SubComponents/Buttons/MaximizeButton/MaximizeButton.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.maximizeButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 7px;
+  background-color: #2f6bf2;
+  border: 1px solid white;
+  color: white;
```
Comment (nico-dev-it) [inline src/Component/Applications/SubComponents/Buttons/MaximizeButton/MaximizeButton.scss]: hex #ffffff

File: src/Component/Applications/SubComponents/Buttons/MinimizeButton/MinimizeButton.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.minimizeButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px;
+  background-color: #2f6bf2;
+  border: 1px solid white;
```
Comment (nico-dev-it) [inline src/Component/Applications/SubComponents/Buttons/MinimizeButton/MinimizeButton.scss:7]: #FFFFFF
same on line 7

File: src/Component/Applications/SubComponents/Header/Header.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  width: 100%;
+  height: 30px;
+  background-color: #0460f6;
+  border: 2px solid #3894ff;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  color: white;
```
Comment (nico-dev-it) [inline src/Component/Applications/SubComponents/Header/Header.scss]: #fffffff

File: src/Component/Applications/AboutApplication/AboutApplication.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.draggable {
+  height: 400px;
+  width: 400px;
+  background-color: white;
```
Comment (nico-dev-it) [inline src/Component/Applications/AboutApplication/AboutApplication.scss]: #ffffff

File: src/Component/Applications/SubComponents/Buttons/CloseButton/CloseButton.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.closeButton {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 10px;
+  background-color: #e1532f;
+  border: 1px solid white;
```
Comment (nico-dev-it) [inline src/Component/Applications/SubComponents/Buttons/CloseButton/CloseButton.scss]: #ffffff

**Review by nico-dev-it — COMMENTED** [review]
just colour code to change #ffffff instead of white

File: src/Component/Applications/SubComponents/Buttons/MaximizeButton/MaximizeButton.scss (hunk 1 above)
Comment (SlothSan) [reply src/Component/Applications/SubComponents/Buttons/MaximizeButton/MaximizeButton.scss]: Done

File: src/Component/Applications/SubComponents/Buttons/MinimizeButton/MinimizeButton.scss (hunk 2 above)
Comment (SlothSan) [reply src/Component/Applications/SubComponents/Buttons/MinimizeButton/MinimizeButton.scss:7]: Done

File: src/Component/Applications/SubComponents/Header/Header.scss (hunk 3 above)
Comment (SlothSan) [reply src/Component/Applications/SubComponents/Header/Header.scss]: Resolved

File: src/Component/Applications/AboutApplication/AboutApplication.scss (hunk 4 above)
Comment (SlothSan) [reply src/Component/Applications/AboutApplication/AboutApplication.scss]: Resolved

File: src/Component/Applications/SubComponents/Buttons/CloseButton/CloseButton.scss (hunk 5 above)
Comment (SlothSan) [reply src/Component/Applications/SubComponents/Buttons/CloseButton/CloseButton.scss]: Resolved

**Pull request merged**